src/gui/layout/qgslayoutviewtoolselect.cpp

```diff
@@ -22,6 +22,9 @@
 #include "qgslayoutitemgroup.h"
 
 
+const int QgsLayoutViewToolSelect::sSearchRadius = 2;
```

Comment:
The issue with this approach is that the layout method is using a search tolerance in layout units, so this is a constant search tolerance regardless of what the current zoom scale of the layout is. Ie if you zoom in, the tolerance will be massive in screen pixels, and if you zoom right out it will become zero. :laughing: 

These tools will need to consider the current layout view transform scale in order to correctly handle this.
You can get the scale via `QgsLayoutView::transform().m11()`

Reply:
> if you zoom right out it will become zero.

Not ideal admitedly 😅. I'll fix that